src/inference/include/ie/ie_parallel.hpp

```diff
@@ -17,6 +17,8 @@
 #include <cstddef>
 #include <type_traits>
 
+#define TBB_PREVIEW_WAITING_FOR_WORKERS 1
```

Comment:
I suppose we need to put it closer to `TBB_PREVIEW_LOCAL_OBSERVER ` with additional guard:
```c++
#ifndef TBB_PREVIEW_WAITING_FOR_WORKERS 
#define TBB_PREVIEW_WAITING_FOR_WORKERS 1
#endif
```